### PR TITLE
Trivia files: fix typos

### PR DESCRIPTION
**Pull request by NCombarieu**
### Description of the changes

Fix multiple Typos in trivia questions/answers

### Have the changes in this PR been tested?
Yes

### CODE REVIEW

File: redbot/cogs/trivia/data/lists/michaeljackson.yaml [hunk 1]
```diff
@@ -39,10 +39,10 @@ AUTHOR: Aurorum
 - Man in the Mirror
 (Identify the Michael Jackson song) At night when the stars shine, I pray in you I'll find a love so true:
 - I Just Can't Stop Loving You
-- I Just Cant Stop Loving You
+- I Just Can't Stop Loving You
```
Comment (japandotorg) [inline redbot/cogs/trivia/data/lists/michaeljackson.yaml:42]: this is not a typo.

File: redbot/cogs/trivia/data/lists/finalfantasy.yaml [hunk 2]
```diff
@@ -833,7 +833,7 @@ AUTHOR: KrisDLuna, Iris, Slackeffect, Falzar
 - Invalid
 - Doesn't have one
 - Non-existent
-- doesnt have one
+- doesn't have one
```
Comment (japandotorg) [inline redbot/cogs/trivia/data/lists/finalfantasy.yaml:836]: not a typo as well

File: redbot/cogs/trivia/data/lists/entertainment.yaml [hunk 3]
```diff
@@ -1347,7 +1347,7 @@ With which period in music do we associate composers such as Beethoven, Mozart a
 With which period in music do we associate composers such as Tchaikovsky, Mendelssohn, and Chopin?:
 - Romantic period
 - Romantic
-With which period in music do we associate with composers such as Bach, Handel and Vivaldi?:
```
Comment (yamikaitou) [inline redbot/cogs/trivia/data/lists/entertainment.yaml:1350]: Original is correct

File: redbot/cogs/trivia/data/lists/general.yaml [hunk 4]
```diff
@@ -650,7 +650,7 @@ The brunch dish of poached eggs, muffins, bacon/ham and hollandaise sauce, popul
 - Benedict
 The caber, kilts and bagpipes are all associated with with European country?:
 - Scotland
-The classical composers Bach, Brahms, Handel, Strauss, and Schumann are all what nationality?:
```
Comment (yamikaitou) [inline redbot/cogs/trivia/data/lists/general.yaml:653]: Original is correct

File: redbot/cogs/trivia/data/lists/entertainment.yaml [hunk 5]
```diff
@@ -458,10 +458,10 @@ This band's highly original video for "Whip it," characterized by red flower pot
 - Devo
 This electronic instrument's creator was surnamed Moog, and his models are worth a fortune! Other brands include Roland, Korg, and Casio?:
 - Synthesizer
-This female artist enjoyed sucess on both popular and country & western stations with such tunes as "Let Me Be There" and "Have You Never Been Mellow"?:
+This female artist enjoyed success on both popular and country & western stations with such tunes as "Let Me Be There" and "Have You Never Been Mellow"?:
 - Olivia Newton-John
 - olivia newton john
-This film starring Julie Andrews and Christopher Plummer wont he best picture Oscar for 1965?:
+This film starring Julie Andrews and Christopher Plummer won't he best picture Oscar for 1965?:
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/entertainment.yaml:464]:
```suggestion
This film starring Julie Andrews and Christopher Plummer won the best picture Oscar for 1965?:
```

File: redbot/cogs/trivia/data/lists/michaeljackson.yaml [hunk 6]
```diff
@@ -39,10 +39,10 @@ AUTHOR: Aurorum
...
 (Identify the Michael Jackson song sung live) Brad, what are you gonna do?:
 - I Just Can't Stop Loving You
-- I Just Cant Stop Loving You
+- I Just Can't Stop Loving You
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/michaeljackson.yaml:45]: Ditto, not a typo.

File: redbot/cogs/trivia/data/lists/overwatch.yaml [hunk 7]
```diff
@@ -161,7 +161,7 @@ On which map was this screenshot taken? http://i.imgur.com/vxqkKQ0.jpg:
 On which map was this screenshot taken? http://i.imgur.com/xuO8PDw.jpg:
 - Volskaya Industries
 - Volskaya
-Orignally when Overwatch was released players could choose to stack multiples of any hero on any team,it was later limited to 1 of each hero per team, and an arcade mode where heros tacking was again possble was added, what is this mode called?:
+Originally when Overwatch was released players could choose to stack multiples of any hero on any team,it was later limited to 1 of each hero per team, and an arcade mode where heroes tacking was again possible was added, what is this mode called?:
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/overwatch.yaml:164]:
```suggestion
Originally when Overwatch was released players could choose to stack multiples of any hero on any team, it was later limited to 1 of each hero per team, and an arcade mode where hero stacking was again possible was added, what is this mode called?:
```

**Review by Flame442 — CHANGES_REQUESTED** [review]
Trivia files are stored in a particular format,
```yaml
Question?:
- Answer 1
- Answer 2
```
The answers often contain differences in formatting (including some slight grammatical inaccuracies, like "dont") in order to make the bot more responsive for players. A few places you changed the _correctly_ incorrect answer, yet that answer should remain.

It looks like you did this automatically, given that and the number of `TwoW ord`s that are incorrectly fixed. In the future, please manually go through your automatically generated output to make sure the sentences still make sense.

Changes requested by other members up to this point also look fine, and are required to satisfy this review.

**Comment by NCombarieu** [comment]
Indeed I used an automatic tool (codespell) to find potential mistakes in these Trivia files.
I am a beginner and would like to learn how to use github and make a contribution to your project to improve myself.
I'll start again from the beginning and try to modify my PR with more attention and let you know very soon.
Thanks for your answer and advice :)

**Comment by Flame442** [comment]
All of the requested changes were never addressed, I believe this has been superseded by #5977, and there's a ton of conflicting files at this point, so I am going to close this PR. Feel free to re-make this PR if there are any remaining typos that were not addressed in that PR.

**Pull request closed without merging**